### PR TITLE
NXP-27543: Diff displays only id for vocabulary fields

### PR DESCRIPTION
**Pull request by RSalem07**

### CODE REVIEW

**Comment by RSalem07** [comment]
The jenkins pipline [see](https://qa.nuxeo.org/jenkins/job/master/job/web-ui-pipeline/240/console) seems to be good, but some test failed i think mainly related to the knonws problem of Document / Note creation

File: elements/diff/elements/nuxeo-user-diff.html [hunk 1]
```diff
@@ -84,12 +84,12 @@
       <template is="dom-if" if="[[_hasObjectInnerChanges(delta)]]">
         <template is="dom-if" if="[[!hideDeletions]]">
           <div class="deletion">
-            <nuxeo-user-tag user="[[delta.id.0]]"></nuxeo-user-tag>
+            <nuxeo-user-tag user="[[originalValue]]"></nuxeo-user-tag>
```
Comment (nmpcunha) [inline elements/diff/elements/nuxeo-user-diff.html]: Thanks for fixing this too.
I would say that it would be more intuitive to:
1. Separate in two different commits;
2. Change the commit message to don't be specific to directories.

Probably I would take the second one. ;)

Comment (RSalem07) [reply elements/diff/elements/nuxeo-user-diff.html]: you welecome, yes i agree to seperate these two commit, i completely forget about doing it
